com.unity.netcode.gameobjects/Runtime/Core/SnapshotSystem.cs

```diff
@@ -651,6 +651,15 @@ public void Dispose()
 
         public void NetworkUpdate(NetworkUpdateStage updateStage)
         {
+            if (m_NetworkManager == null)
+            {
+                Debug.Log("m_NetworkManager is null");
+            }
+            else if (m_NetworkManager.NetworkConfig == null)
+            {
+                Debug.Log("m_NetworkManager.NetworkConfig is null");
+            }
```

Comment:
I assume this is for testing/debugging but not for actual PR?

Reply:
Oh darn, yes, let me remove that.